webapp/src/sidepanel.tsx

```diff
@@ -153,6 +155,11 @@ export class Sidepanel extends data.Component<SidepanelProps, SidepanelState> {
                         {showFileList && <filelist.FileList parent={parent} />}
                         {showFullscreenButton && <div id="miniSimOverlay" role="button" title={lf("Open in fullscreen")} onClick={this.handleSimOverlayClick} />}
                     </div>
+                    {isTabTutorial && <div className="tutorial-controls">
+                        { backButton }
+                        <Button icon="lightbulb" disabled={true} className="tutorial-hint" />
```

Comment:
is it worth showing this button if it's always disabled?

Reply:
yeah, technically all three of these are fake buttons (the back and next directly navigate you to the tutorial tab and don't change the step), the thought was that users are familiar with the three buttons at the bottom, so we shouldn't change the positioning/text/etc because a new kind of button might still be confusing